discovery-scripts/nix/smartctl-disks-discovery.pl

```diff
@@ -50,7 +50,8 @@
     }
 }
 else {
-    for (`$smartctl_cmd --scan-open`) {
+    for (`$smartctl_cmd --scan-open`.
+         `$smartctl_cmd --scan-open -dnvme`) {
```

Comment:
I tested this PR on top of master, as master does not correctly detect SMART_ENABLED on nvme.

It seems to works correctly if I remove this `--scan-open -dnvme` line (and nvme is also detected later with `nvme`). 
Also tested on SSD, and it works.
This seems to be the same behavior as in #85 .

Reply:
yes, it should work with nvme cli even without this PR. So no point reverting this. But the whole idea is to discover `nvme` drives using only `smartctl`